talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -377,6 +377,10 @@ void P2PClient::OnMessageReceived(const std::string& remote_id,
                          remote_id, message);
 }
 void P2PClient::OnStopped(const std::string& remote_id) {
+  {
+    const std::lock_guard<std::mutex> removed_lock(removed_pc_mutex_);
```

Comment:
Add a comment about why we need the graveyard: "Need to keep a reference around to the peer connection channel to avoid deleting it while its executing an object method. The next OnStopped will then delete the object. In the mean time, the underlying peer connection will still be closed"